android/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
     'java'               : JavaVersion.VERSION_11,
     'androidGradlePlugin': '7.4.2',
     'googleServices'     : '4.3.10',
-    'voiceAndroid'       : '6.6.1',
+    'voiceAndroid'       : '6.6.2',
```

Comment:
6.6.3 is now the latest version, it supports android 15

Reply:
Thanks for the heads-up. I'll pull `6.6.3` when I work on confirming the Android 15 support. Is that the only change between `6.6.2` and `6.6.3`?